content/bn/virtual-machine.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ভার্চুয়াল মেশিন
+status: Feedback Appreciated
+category: প্রযুক্তিবিদ্যা
+---
+
+## এটা কি
+
+ভার্চুয়াল মেশিন (VM) হল একটি কম্পিউটার এবং এর অপারেটিং সিস্টেম যা একটি নির্দিষ্ট হার্ডওয়্যারের সাথে আবদ্ধ নয়। একটি ফিজিক্যাল কম্পিউটারে, একাধিক ভার্চুয়াল কম্পিউটার বানাতে ভিএমগুলি ভার্চুয়ালাইজেশনের উপর নির্ভর করে। এই বিচ্ছেদটি সংস্থা এবং পরিকাঠামো প্রদানকারীদের অন্তর্নিহিত হার্ডওয়্যারকে প্রভাবিত না করে সহজেই ভিএম তৈরি এবং ধ্বংস করতে দেয়।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ভার্চুয়াল মেশিন ভার্চুয়ালাইজেশনের সুবিধা নেয়। যখন একটি বেয়ার-মেটাল মেশিন একটি একক অপারেটিং সিস্টেমের সাথে আবদ্ধ থাকে, তখন মেশিনের সম্পদগুলি কতটা ভালভাবে ব্যবহার করা যেতে পারে তা কিছুটা সীমিত। এছাড়াও, যখন একটি অপারেটিং সিস্টেম একটি একক ফিজিক্যাল মেশিনের সাথে আবদ্ধ থাকে, তখন এর প্রাপ্যতা সরাসরি সেই হার্ডওয়্যারের সাথে সংযুক্ত থাকে। রক্ষণাবেক্ষণ বা হার্ডওয়্যার ব্যর্থতার কারণে যদি ফিজিক্যাল মেশিন অফলাইনে থাকে, তাহলে অপারেটিং সিস্টেমও অফলাইন হয়ে পরে।
+
+## এটা কিভাবে সাহায্য করে
+
+একটি অপারেটিং সিস্টেম এবং একটি একক ফিজিক্যাল মেশিনের মধ্যে সরাসরি সম্পর্ক অপসারণ করলে, বেয়ার-মেটাল মেশিনের বেশ কয়েকটি সমস্যা সমাধান হয়: প্রভিশনিং টাইম, হার্ডওয়্যার ইউটিলাইজেশন এবং রেজিলেন্সি। এটি সমর্থন করার জন্য কোন নতুন হার্ডওয়্যার কেনা, ইনস্টল করা বা কনফিগার করা না থাকায়, একটি নতুন কম্পিউটারের জন্য প্রভিশনিং টাইম নাটকীয়ভাবে উন্নত হয়েছে। ভিএমগুলি আপনাকে একটি একক ফিজিক্যাল মেশিনে একাধিক ভার্চুয়াল মেশিন স্থাপন করে আপনার বিদ্যমান ফিজিক্যাল হার্ডওয়্যারের সম্পদগুলিকে আরও ভালভাবে ব্যবহার করার অনুমতি দেয়। একটি নির্দিষ্ট ফিজিক্যাল মেশিনের সাথে আবদ্ধ নয়, ভিএমগুলিও ফিজিক্যাল মেশিনের চেয়ে বেশি স্থিতিস্থাপক। যখন একটি ফিজিক্যাল মেশিনের অফলাইনে যেতে হয়, তখন এতে চলমান ভিএমগুলিকে অন্য মেশিনে স্থানান্তরিত করা যেতে পারে যেখানে সামান্য বা কোন ডাউনটাইম নেই।
```

Comment:
Can you make a much more simple sentence instead of this `একটি নির্দিষ্ট ফিজিক্যাল মেশিনের সাথে আবদ্ধ নয়, ভিএমগুলিও ফিজিক্যাল মেশিনের চেয়ে বেশি স্থিতিস্থাপক।` ?

Reply:
Ok @mitul3737, I can make the sentences better. Let me rewrite those.